### PR TITLE
fix #212 adding a new tool

### LINKED ISSUE #212: tools: open peeps
A hand-drawn illustration library:
https://openpeeps.com/

## Acceptance criteria

- [ ] Add tool - https://github.com/schalkneethling/developer-toolchest#tools
- [ ] Add logo - https://github.com/schalkneethling/developer-toolchest#tools

### PR DESCRIPTION
**Pull request by Rico-Smit**
fix #212 Open Peeps have been added

### CODE REVIEW

File: public/tools.json [hunk 1]
```diff
@@ -1518,5 +1518,14 @@
     "logo": "feather-logo.png",
     "alt": "",
     "tag": ["fonts", "typography", "icon", "icons", "iconography"]
+  },
+  {
+    "id": 162,
+    "title": "Open Peeps",
+    "url": "https://openpeeps.com/",
+    "description": "Hand-drawn illustrations of people of the world. Can you recreate yourself?",
+    "logo": "openpeeps.png",
```
Comment (schalkneethling) [inline public/tools.json:1527]: I do not see this image added to the repo? Did you perhaps forget to check it into source control?

**Review by schalkneethling — CHANGES_REQUESTED** [review]
Thank you for the contribution, @Rico-Smit 🎉 It looks like you forgot to add the image file to source control. Other than that, this looks good.

File: public/tools.json (hunk 1 above)
Comment (schalkneethling) [reply public/tools.json:1527]: @Rico-Smit a friendly nudge.

**Comment by schalkneethling** [comment]
@Rico-Smit A couple of things:

1. There is a conflict in `tools.json` - most likely because another PR was merged that uses the same `id` as this one. If you bump your `id` up, all should be good.
2. Please add `fix #212` to your pull request description so that GitHub can link the issue and the pull request.
3. The image you reference seems to be missing from the pull request

Thanks!

File: public/tools.json (hunk 1 above)
Comment (Rico-Smit) [reply public/tools.json:1459]: @schalkneethling  The img is attached and the conflict is resolved.
open-peeps

**Comment by schalkneethling** [comment]
Hey @Rico-Smit, thank you for addressing the merge conflict. It looks like you missed points two and three of my comment here: https://github.com/schalkneethling/developer-toolchest/pull/246#issuecomment-1429827865

**Review by schalkneethling — APPROVED** [review]

**Review by schalkneethling — APPROVED** [review]

**Review by schalkneethling — APPROVED** [review]

**Pull request merged**